crates/core/src/db/relational_db.rs

```diff
@@ -84,7 +93,11 @@ impl RelationalDB {
     ///
     /// The [`tokio::runtime::Handle`] is used to spawn background tasks which
     /// take care of flushing and syncing the log.
-    pub fn local(root: impl AsRef<Path>, rt: tokio::runtime::Handle, address: Address) -> Result<Self, DBError> {
+    pub fn local(
+        root: impl AsRef<Path>,
+        rt: tokio::runtime::Handle,
+        address: Address,
+    ) -> Result<(Self, ConnectedClients), DBError> {
```

Comment:
Can you add to the docs describing the return values? Github won't let me drop a suggestion on an unchanged line, but I'd add a paragraph reading:

```rs
/// Returns the set of clients who were connected as of the most recent transaction
/// as a [`ConnectedClients`]
/// so that the newly-restored module can call disconnect for each of them.
```